This patch contains the changes done in the release branch.
The patch was generated with the following command:

svn diff -r{20090303} svn://svn.mplayerhq.hu/ffmpeg/branches/0.5

It can be dropped on the next upstream upgrade

--- a/version.sh
+++ b/version.sh
@@ -1,20 +1,6 @@
 #!/bin/sh
 
-# check for SVN revision number
-revision=$(cat snapshot_version 2> /dev/null)
-test $revision || revision=$(cd "$1" && LC_ALL=C svn info 2> /dev/null | grep Revision | cut -d' ' -f2)
-test $revision || revision=$(cd "$1" && grep revision .svn/entries 2>/dev/null | cut -d '"' -f2)
-test $revision || revision=$(cd "$1" && sed -n -e '/^dir$/{n;p;q}' .svn/entries 2>/dev/null)
-test $revision && revision=SVN-r$revision
-
-# check for git short hash
-if ! test $revision; then
-    revision=$(cd "$1" && git log -1 --pretty=format:%h)
-    test $revision && revision=git-$revision
-fi
-
-# no version number found
-test $revision || revision=UNKNOWN
+revision=0.5
 
 test -n "$3" && revision=$revision-$3
 
